unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs

```diff
@@ -1,20 +1,19 @@
+using Cysharp.Threading.Tasks;
 using DCL;
+using DCL.Emotes;
 using DCL.EmotesCustomization;
 using DCL.Helpers;
 using DCL.Interface;
 using DCL.NotificationModel;
+using DCL.Tasks;
+using DCLServices.WearablesCatalogService;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading;
-using Cysharp.Threading.Tasks;
-using DCL.Emotes;
-using DCL.Tasks;
-using DCLServices.WearablesCatalogService;
 using UnityEngine;
 using UnityEngine.Rendering;
 using UnityEngine.Rendering.Universal;
-using Categories = WearableLiterals.Categories;
```

Comment:
Why did you remove this? I think it made the code more understandable putting
```
Categories.BODY_SHAPE, Categories.UPPER_BODY, Categories.LOWER_BODY...
```
instead of
```
WearableLiterals.Categories.BODY_SHAPE, WearableLiterals.Categories.UPPER_BODY, WearableLiterals.Categories.LOWER_BODY
```

Reply:
Auto code cleanup from the IDE, will rollback